bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -335,6 +357,18 @@ func LoadKfAppCfgFile(cfgfile string) (kftypesv3.KfApp, error) {
 		}
 	}
 
+	// Since we know we have a local file we can set a default name if none is set based on the local directory
+	if kfdef.Name == "" {
+		kfdef.Name = nameFromAppFile(appFile)
+		if kfdef.Name == "" {
+			return nil, &kfapis.KfError{
+				Code:    int(kfapis.INTERNAL_ERROR),
```

Comment:
shouldn't this be `INVALID_ARGUMENT`?  from the codes there are 2 error cases:
- not able to fetch absolute path.
- appFile lives in root directory.

seems to me both of the cases are error use of the CLI?  also root cause is `Spec.Name` is not set and we have to infer from the path, which is also on users?